pkgs/applications/misc/ArchiSteamFarm/default.nix

```diff
@@ -5,48 +5,51 @@
 , libkrb5
 , zlib
 , openssl
+, callPackage
+, stdenvNoCC
 }:
 
 buildDotnetModule rec {
   pname = "archisteamfarm";
-  version = "5.1.5.3";
+  version = "5.2.0.10";
 
   src = fetchFromGitHub {
     owner = "justarchinet";
     repo = pname;
     rev = version;
-    sha256 = "sha256-H038maKHZujmbKhbi8fxsKR/tcSPrcl9L5xnr77yyXg=";
+    sha256 = "sha256-okI58EpzKPd9IfC0K687inscmhO74vMeq8crEF/Xj3k=";
   };
 
-  dotnet-runtime = dotnetCorePackages.aspnetcore_5_0;
-  nugetDeps = ./deps.nix;
+  dotnet-runtime = dotnetCorePackages.aspnetcore_6_0;
+  dotnet-sdk = dotnetCorePackages.sdk_6_0;
+
+  nugetDeps = if stdenvNoCC.isAarch64 then ./deps-aarch64.nix else ./deps.nix;
```

Comment:
Are the aarch64 deps fetchable from an x86 host? Updating this would get much less accessible if that's not the case.

Reply:
I've changed the update script to rely on binfmt to pull in the dotnet for all required architecture, which output the correct dependcies files. The downside of this is that it relies on binfmt + qemu being set up correctly. Although it can easily be enabled, on nixos, with `boot.binfmt.emulatedSystems = [ "aarch64-linux" ];`